src/dotnet/commands/dotnet-new/CSharp_Lib/project.json.template

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc3-24126-00"
+  },
+  "frameworks": {
+    "netstandard1.5": {}
```

Comment:
This should be netstandard1.6 and it should match the version of the package (once the 1.6 version is ready).